util/test/sub_test

```diff
@@ -202,6 +202,105 @@ def LauncherTimeoutArgs(seconds):
               useLauncherTimeout)
 
 
+# Execute command.
+# Its stdout+stderr go to sub_test's stdout+stderr.
+# Return its exit code.
+def runAndShowOutput(command):
+    sys.stdout.flush()
+    p = subprocess.Popen(command)
+    rc = p.wait()
+    return rc
+
+# Execute command. Rc = Return code.
+# Gather its combined stdout+stderr.
+# Return (its exit code, the gathered output).
+def runAndReturnRcOutput(command):
+    p = subprocess.Popen(command,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.STDOUT)
+    output = p.communicate()[0]
+    return p.returncode, output
+
+# Execute command. EIEO = Error If Error Output.
+# Gather *separately* its stdout and stderr.
+# If stderr is non-empty or the exit code is non-0, report an error.
+# If errfatal, quit with Fatal(). Otherwise, 
+# return the return code (at least 1 if got stderr) and command's stdout.
+def runAndReturnRcOutputEIEO(command, errfatal=False):
```

Comment:
How do you all feel about mostly not changing the behavior upon this first PR, then following up with behavior changes separately?

Meanwhile let us choose good names right away. We could leave non-good names for functions that we expect to retire, if we can't come up with good ones.